contracts/src/allocator/Allocator.sol

```diff
@@ -38,12 +38,14 @@ contract Allocator is Pausable, UsingConfig, IAllocator, UsingValidator {
 	);
 
 	uint64 public constant basis = 1000000000000000000;
+	uint64 public constant wait_allocate = 11520;
```

Comment:
@Akira-Taniguchi Isn't it better to be able to update `wait_allocate` from outside? Block times are variable, with an average of 13~14 seconds.

https://etherscan.io/chart/blocktime

Reply:
and, how about naming this one with `wait_until_allocatable`?

Reply:
Thank you for pointing this out.
Corrected.